example/src/services/BLEService/BLEService.ts

```diff
@@ -280,11 +280,11 @@ class BLEServiceInstance {
       this.showErrorToast(deviceNotConnectedErrorText)
       throw new Error(deviceNotConnectedErrorText)
     }
-    return this.manager.isDeviceConnected(this.device.id).catch(error => {
-      this.onError(error)
-    })
+    return this.manager.isDeviceConnected(this.device.id)
   }
 
+  isDeviceWithIdConnected = (id: string) => this.manager.isDeviceConnected(id).catch(console.error)
```

Comment:
It's good practice to infer the type, via `Device['id']`

Reply:
already fixed in https://github.com/dotintent/react-native-ble-plx/pull/1111